changes/ce/fix-10107.en.md

```diff
@@ -0,0 +1,7 @@
+For operations on `bridges API` if `bridge-id` is unknown we now return `404`
+instead of `400`. Also a bug was fixed that caused a crash if that was a node
+operation. Additionally we now also check for cluster operation `start` if given
+bridge is enabled. Affected endpoints:
```

Comment:
Personal: Additionally we now also check if the given bridge is enabled when doing the cluster operation `start` .